api/interfaces/pinia_testing.TestingOptions.md

```diff
@@ -10,88 +10,86 @@ sidebarDepth: 3
 
 [@pinia/testing](../modules/pinia_testing.md).TestingOptions
 
-## Properties
+## Propiedades
 
 ### createSpy
 
 • `Optional` **createSpy**: (`fn?`: (...`args`: `any`[]) => `any`) => (...`args`: `any`[]) => `any`
 
-#### Type declaration
+#### Declaración del tipo
```

Comment:
Tengo dudas si debe de ser "Declaración del tipo" o declaración del tipado"

Reply:
"Declaración del tipado" lo veo más entendible para el resto de usuarios

Reply:
Yo lo traduciría más bien como "Tipado de la declaración", ya que realmente lo que haces es tipar la declaración de la interfaz/función en vez de declarar los tipos para ello. ¿Cómo lo ves?

Reply:
Me parece excelente de esa manera! Changing it right away